src/app/content/highlights/components/HighlightsPopUp.tsx

```diff
@@ -123,6 +124,7 @@ class HighlightsPopUp extends Component<Props> {
           </Styled.Header>
           {this.props.user ? <ShowMyHighlights /> : this.loginForHighlights()}
         </Styled.Modal>
+        {this.props.user && <HighlightsHelpInfo />}
```

Comment:
you already have a conditional `return null` inside here, it might be cleaner to just check for user there as well